server/app/services/applicant/ApplicantService.java

```diff
@@ -819,32 +819,30 @@ public CompletionStage<ApplicationPrograms> relevantProgramsForApplicant(
             .getApplicationsForApplicant(
                 applicantId, ImmutableSet.of(LifecycleStage.DRAFT, LifecycleStage.ACTIVE))
             .toCompletableFuture();
-    CompletableFuture<ImmutableList<ProgramDefinition>> activeProgramDefinitionsFuture =
-        userRepository
```

Comment:
**thought:** Doesn't seem like we are doing anything with the applicant info here, so should be able to remove this? Unless I'm missing something?

Reply:
Great, thank you!